storage/stowstore.go

```diff
@@ -36,6 +37,10 @@ var fQNFn = map[string]func(string) DataReference{
 	azure.Kind: func(bucket string) DataReference {
 		return DataReference(fmt.Sprintf("afs://%s", bucket))
 	},
+	local.Kind: func(bucket string) DataReference {
+		// Eventually we may want to make this file://
```

Comment:
I don't understand this comment, is that not the case below?

Reply:
aah older comment, should delete it actually

Reply:
done